tools/analyzer_plugin/tool/dart_dev/config.dart

```diff
@@ -3,6 +3,11 @@ import 'package:glob/glob.dart';
 
 final config = {
   ...coreConfig,
+  'analyze': AnalyzeTool()
+    ..include = [
+      Glob('lib/**.dart'),
+      Glob('tool/**.dart'),
+    ],
```

Comment:
Do we need to limit the includes like this? I think the `analysis_options.yaml` should handle excluding the `playground/` subdir, so analyzing the rest of the dart files in `tools/analyzer_plugin/` should work 🤞

Reply:
Ah yep, good call! Addressed that!